public/admin/config.yml

```diff
@@ -798,7 +798,7 @@ collections:
     files:
       - name: Info tooltips
         label: Info tooltips
-        file: src/cms-content/info-tooltips.json
+        file: src/cms-content/tooltips/info-tooltips.json
```

Comment:
re-did the original setup, in which i forgot to name a folder and so the files went directly into `cms-content`-- hence the deleted files below (`src/cms-content/info-tooltips.json`, `src/cms-content/metric-calculation-tooltips.json`)

i also renamed the outer folder from `cms-content/infoTooltips` --> `cms-content/tooltips`, hence the deleted `src/cms-content/infoTooltips/index.ts`

😬 😬